pages/common/xh.md

```diff
@@ -0,0 +1,28 @@
+# xh
+
+> XH: Friendly and fast tool for sending HTTP requests.
+> More information: <https://github.com/ducaale/xh>.
+
+- Send a GET request:
+
+`xh {{httpbin.org/get}}`
+
+- Send a POST request with JSON body: `{"name": "ahmed", "age": 25}`:
```

Comment:
```suggestion
- Send a POST request with JSON body:
```

Reply:
I think that may be useful to visualize how JSON will look like

Reply:
What about this, then?
```suggestion
- Send a POST request with JSON body: (each key specified is added to the top-level e.g. `{"name": "john", "age": 25}`):
```